Example/Podfile

```diff
@@ -1,6 +1,3 @@
-#source 'https://github.com/CocoaPods/Specs.git'
-#source 'https://github.com/primer-io/primer-podspecs.git'
```

Comment:
Is there a particular reason to remove the cocoapods main source too?

Reply:
When you don't want to add a private repo, there's no need to add the cocoapods trunk, it defaults to this. And we don't the private repo anymore, 3DS became public.